conduct.md

```diff
@@ -24,15 +21,14 @@ As a member of the community, you agree to the following:
 
 **Encouraged:**
 
-- Be kind and courteous.
-- Respect differences of opinion and remember that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
-- Remember that everyone was new to Scala at some point. We want to encourage newcomers to join our community and learn the Scala language and ecosystem. Assume competence.
-- Show empathy towards other community members.
```

Comment:
What's the reason to drop "Show empathy towards other community members."?

Reply:
It felt like a duplication of the above points, now that they spell out what kindness and respect entail. If that is not the case, then please suggest contents for an accompanying paragraph as for the others.

Reply:
I’ll add “empathy” to the first bullet point explicitly, then I feel that we definitely have it covered.